targets/web/src/elements.ts

```diff
@@ -1,5 +1,4 @@
-export type JSXElements = keyof JSX.IntrinsicElements
-export const elements: JSXElements[] = [
+export const elements: Array<keyof JSX.IntrinsicElements> = [
```

Comment:
This seems like an unnecessary change. Let's keep PRs focused.

Reply:
> fixed a web package build problem ([!] Error: Character is out of bounds)

Were you seeing this error when run `yarn build` or something?

> if someone was really wanting something like JSXElements, it's likely they shouldn't pull this from this package.

The `JSXElements` type isn't re-exported, so I don't think that's an issue.